File: 2021 스터디 1주차/2021 스터디 1주차/1193.c
#include<stdio.h>

int main() {
	int layer = 1, son, mom;
	int num = 0;

	scanf("%d", &num);

	while (num > layer) {
		num -= layer;
		layer++;
	}
	if (layer % 2 == 0) {
		mom = layer - num + 1;
		son = num;
	}else {
		son = layer - num + 1;
		mom = num;
	}
	printf("%d/%d", son, mom);
}

File: baekjun-2021study/2021 스터디 1주차 파이썬/1193.py
layer = 1
num = int(input())

while num > layer:
    num -= layer
    layer += 1

if layer % 2 == 0:
    mom = layer - num + 1
    son = num
else:
    son = layer - num + 1
    mom = num

result = str(son)+"/"+str(mom)

print(result)